sessions for self-managed dbms docs (manual + notebook) (#719)
neo4j/graph-data-science-client: doc/modules/ROOT/pages/tutorials/gds-sessions.adoc

== Creating a new session

A new session is created by calling `sessions.get_or_create()`. To do
this, we need to identify a data source. We assume here that an AuraDB
instance has been created and is available for access. We need to pass
the database address, username and password to the `DbmsConnectionInfo`
class.

We also need to specify the size of the session. There are many possible
sizes to choose from. Please refer to the API reference documentation or
the manual for a full list.

Finally, we need to give our session a name. We will call ours

> `people-and-fruits’. If we have already created a session and want to
> reconnect to it, the same code will be used. There is no extra cost in
> doing this, and it is much faster.

Reviewer: ```suggestion
`people-and-fruits’. It is possible to reconnect to an existing session by calling `get_or_create` with the same session name and configuration. 
```
